packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -473,6 +489,8 @@ function updateHostComponent(
       // Note that this might release a previous clone.
       workInProgress.stateNode = currentInstance;
       return;
+    } else {
+      markCloned(workInProgress);
```

Comment:
is this right? seems like we should only mark clone if `requiresClone` is true

Reply:
`cloneInstance` above returned a new instance since `newInstance !== currentInstance` in this branch, so the cloned tree needs to bubble up (I think)